### PR TITLE
Deploy to quay on tags

### PR DESCRIPTION
**Pull request by pivnicek**
Required for https://keboola.atlassian.net/browse/PS-58

### CODE REVIEW

File: deploy.sh [hunk 1]
```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+set -e
+
+docker login -u="$QUAY_USERNAME" -p="$QUAY_PASSWORD" quay.io
+docker tag keboola/docker-jupyter quay.io/keboola/provisioning-client:${TRAVIS_TAG}
```
Comment (ondrejhlavacek) [inline deploy.sh]: docker-jupyter? :-) you can try this out when you remove the tags condition. push to this branch should execute the push. anyway why quay? is it used somewhere public?

Comment (pivnicek) [reply deploy.sh]: tvl, sorry!!

Comment (pivnicek) [reply deploy.sh]: quay because I thought we liked to have our clients on quay.

Comment (ondrejhlavacek) [reply deploy.sh]: I guess so, but it's a pain when quay is down (happens from time to time)

Comment (ondrejhlavacek) [reply deploy.sh]: just realized - it's a client, not a cli. this does nothing in a docker container.

Comment (ondrejhlavacek) [reply deploy.sh]: or what is your usecase? if you don't need it anywhere, don't create it :-)

Comment (pivnicek) [reply deploy.sh]: Sorry @ondrejhlavacek I should have mentioned in the description.  This is to bring it inline with how we do the others.  Please see https://github.com/keboola/provisioning-router/pull/186#issuecomment-555003465

Comment (ondrejhlavacek) [reply deploy.sh]: I see, thanks, makes sense.

**Comment by ondrejhlavacek** [comment]
Sorry for my failed build, tagged the commit and then figured out it's already running a build, so I deleted the tag but cancelled the build. Anyway your build is failing too - https://travis-ci.com/keboola/provisioning-client/builds/137011278

So I'll wait until the images are in the quay repo

**Comment by pivnicek** [comment]
I'm sorry for the pre-mature review request.   your response time is 🔥 :-)

**Comment by pivnicek** [comment]
So @ondrejhlavacek sorry for the delay here, I had to fix provisioning-router for the tests to pass.  Everything finally looks ok to go ahead with this now imho.

**Pull request merged**